macros/core/PGbasicmacros.pl

```diff
@@ -316,7 +316,7 @@ sub NAMED_ANS_RULE {
 	$tcol = $tcol < 40 ? $tcol : 40;           ## get min
 
 	MODES(
-		TeX        => "\\answerRule{$tcol}",
+		TeX        => ($name ? "\\answerRule[$name]{$tcol}" : "\\answerRule{$tcol}"),
```

Comment:
The `$name ? ...` checking is pointless.  `$name` will always be set.  All answers are actually labeled answers internally for PG.  So this should be changed to
`TeX => "\\answerRule[$name]{$tcol}"`.  That is what will always happen anyway.  The same for the rest of the instances in this file.

Reply:
OK, I changed those.